### ISSUE TITLE
Transfer to `terra-farm` organisation?

### ISSUE THREAD

**Issue by ringods** [riobard/go-virtualbox#6]
Hello @riobard,

The `terra-farm` github organisation is a structured attempt to have a community around Terraform providers, the API projects used in the providers as well as Terraform modules. 
https://github.com/terra-farm/
One of the aims is to have more than 1 maintainer on a repository.

A fork of this repository is used as the Go API for the Terraform VirtualBox provider.
https://github.com/terra-farm/terraform-provider-virtualbox

Would you mind transferring this repo to the `terra-farm` organisation for further maintenance?

My intent is to ask all fork owners to submit pull requests to this repo and see who of the fork owners would like to contribute to this repository.

**Comment by riobard** [comment]
What's wrong with the fork?

**Comment by ringods** [comment]
There are a number of forks, each with it's own improvements. I would like to benefit from all of these. Given the open pull requests in your original repo are already from 2015, I have the impression (no offense) that this repo is unmaintained. Hence my request for the transfer to a community where more than 1 person can help out.

**Comment by riobard** [comment]
Yeah this repo is unmaintained for quite a while and there are forks that are actively maintained. I'm wondering why you don't just fork yourself and ask the other forks for PR. Basically if I later want to contribute to it I'll have to fork another which seems pointless.

**Comment by ringods** [comment]
@riobard transfer to the organisation doesn't mean you loose your write access to the repository. Sorry if that was not clear. 

With your repo as the root of the whole network, it is just easier to perform all these actions. When transferring, Github will also redirect web views from github.com/riobard/go-virtualbox to github.com/terra-farm/go-virtualbox.

It's just a drop easier...

**Comment by riobard** [comment]
Ah now I see what you want. There're a couple of things missing anyway (like choosing a proper license, etc), and I'm considering it. But first of all… who are you guys and what's your relationship with HashiCorp? 😄

**Comment by ringods** [comment]
Well, we are only a handful of people just contributing to either third-party providers or specific modules for Terraform. With "third-party" I mean that such providers are not maintained by the core Hashicorp team. The idea is to create a community that keep an eye on shared maintenance of all the code under its wings. You can compare it to similar communities for:

* Chef cookbooks: [Sous-Chefs](https://github.com/sous-chefs)
* Puppet modules: [Vox Pupuli](https://github.com/voxpupuli)

We started the initiative because we noticed a number of third-party providers already spread around with various levels of maintenance.

For the rest, we are not affiliated with Hashicorp in any specific way. Sometimes we ask Hashicorp for resources, like they created an additional chat channel for us on their [Terraform gitter community](https://gitter.im/hashicorp-terraform/terra-farm).

**Comment by ringods** [comment]
Hello @riobard, did you have time to review my last comment? Let me know if you need more information regarding my request. Tnx.

**Comment by riobard** [comment]
Ah, sorry about that. Almost forgot :P 

What license do you want to use for this repo?

**Comment by ringods** [comment]
You already added the Apache 2.0 License on 23 April. See commit 39b919eb87081c0dbf971b3a86c62c288facf1e8. I think that is OK. The virtualbox provider is MIT licensed and can work together with the Apache 2.0 license if I'm not mistaken.

**Comment by riobard** [comment]
Ah, right! I did that :D  Now everything is set… Just tried to transfer the repo to your org but GitHub complained
> You don’t have the permission to create repositories on terra-farm

**Comment by ringods** [comment]
@riobard since I'm interested in co-maintaining this repo, can you add me as admin to this repo? I am also an admin of the `terra-farm` organization so I can execute the transfer.

**Comment by riobard** [comment]
@ringods Invitation sent.

**Comment by ringods** [comment]
@riobard Invitation acknowledged. Can you make me a repo admin now? I need to be able to go into the repo settings to initiate the transfer.

**Comment by riobard** [comment]
@ringods How do I do that? I couldn't find the settings anywhere.

**Comment by ringods** [comment]
@riobard my bad. I have been working with Github organisations for too long already: repos under an organization have 3 levels of access: admin, write, read. That doesn't seem to be the case for user repositories.

I'll make your organization owner temporarily so you can transfer the ownership.

**Comment by riobard** [comment]
@ringods GitHub still complained 

> You don’t have the permission to create repositories on terra-farm.

**Comment by ringods** [comment]
@riobard did you receive an email invitation to become member of the `terra-farm` organization? If so, did you accept the invitation? I'm asking because I don't see you listed in the list of members.

**Comment by riobard** [comment]
@ringods Ah my bad. It's now done. Transfer in progress. Should be in the org repo soon.

**Comment by riobard** [comment]
Cool!
